roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -675,3 +675,6 @@ sysctl_file_path: "/etc/sysctl.d/99-sysctl.conf"
 
 system_upgrade: false
 system_upgrade_reboot: on-upgrade  # never, always
+
+# Forensic container checkpointing in Kubernetes. It's experimental so far. See [Checkpoint API](https://kubernetes.io/docs/reference/node/kubelet-checkpoint-api/)
+container_checkpoint_enabled: false
```

Comment:
Because the var is only work for cir-o, it's better to change the var to `crio_criu_support_enabled` instead `container_checkpoint_enabled`, it make change more smaller.

And using docs to tell users to open the feature gates.

Reply:
Thanks. Now changes are smaller, and more focused on cri-o.